File: Exercicios_Em_Python_CV/desafio_008.py
#Escreva um programa que leia um valor em metros e o exiba convertido
#em centimetros e milímetros
metros = float(input('Digite o total de metros a ser convertido:'))
print('Total em metros:{:.2f}m'.format(metros))
print('Total em centímetros:{:.2f}cm'.format(metros*100))
print('Total em milímetros:{:.2f}mm'.format(metros*1000))

File: Exercicios_Em_Java_CV/Desafio008.java
package fundamentos;

import java.util.Scanner;

/*
 * Escreva um programa que leia um valor em metros e o exiba
 * convertido em centimetros e milimetros
 */
public class Desafio008 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.print("Digite um valor em metros:");
		float metros = scanner.nextFloat();
		System.out.println("Valor em centimetros:" + metros * 100);
		System.out.println("Valor em milimetros:" + metros * 1000);
		scanner.close();
	}
}
